src/platform/esp32/architecture.h

```diff
@@ -87,6 +87,7 @@
 #define HW_VENDOR meshtastic_HardwareModel_TLORA_V2_1_1P8
 #elif defined(T_DECK)
 #define HW_VENDOR meshtastic_HardwareModel_T_DECK
+#define USE_PERSISTENT_MSG
 #elif defined(T_WATCH_S3)
```

Comment:
This define is best moved to the variant file if you really want it only for the t-deck.

Reply:
@caveman99 Would you prefer to try implement it everywhere? I'll withdraw my objections if you do think it'd be better.

Reply:
I second this, the Tdeck UI will be out someday, but all other devices will be left behind, this is the best option we've gotten so far. Maybe have a new module and the ability to give the choice of turning this on or not.

Reply:
My idea initially was to maybe get it released for T-Deck first, and add other devices once they're tested. There's definitely a lot of edge case stuff that would need to be handled, and some devices probably shouldn't get this feature at all imo, but if the devs want to go ahead and try, I'll support the effort.